src/main/java/com/techsophy/tsf/runtime/form/constants/FormModelerConstants.java

```diff
@@ -264,4 +264,5 @@ public class FormModelerConstants
     public static final String RUNTIME_FORM = "tp-app-runtime-form";
     public static final String RUNTIME_FORM_MODELER_API_VERSION_1 = "Runtime Form API v1.0";
     public static final String SERVICE = "service";
+    public static final String DATABASE_NAME = "${database.name}";
```

Comment:
set the default to techsophy-platform

Reply:
dthis change is required, I asked to initialize with default if database.name is not already defined.
${database.name:techsophy-platform}